packages/utils/logger/test/index.ts

```diff
@@ -1,5 +1,5 @@
 import 'mocha'
-import createLogger, { Logger } from '../src'
+import createLogger, { Logger } from '@interledger/utils-logger'
```

Comment:
I think one thing worth pointing out is there seemed to be inconsistent imports ... using the mocha ts-node/register resolution and some via the `require('..')` package.json#main -> build/index.js jazz

Reply:
Probably some legacy stuff there.
We should def favour consistency